src/globals/_expands.scss

```diff
@@ -0,0 +1,16 @@
+/*
+* Percentages for component expansion
+* For percentage 0, a default value of 140px was defined
+*/
+
+$exp_0: 140px;
```

Comment:
Please, remove this variables. The width can be the minor (140px default) and bigger with 100%.